#20 Add class description to display of DataFrame

mreijnde opened the pull request:
Minor change to add class to display of DataFrame and show if it is a series or not.

closes #20

benjamingaudin commented on +frames/DataFrame.m:
```diff
@@ -1380,7 +1380,20 @@ function disp(obj)
+                    % show content
+                    disptable = obj.t;
+                    if obj.colseries
+                       disptable.Properties.VariableNames{1} = 'NaN';
```
'missing' instead of 'NaN' as the value could be a NaT, a string(missing) as well

benjamingaudin commented:
What I mean is: showing ‘NaN’ will make the user believe that the type of the missing value is a double (or a duration); while the type of the missing value could be a datetime (NaT) or a string (<missing>). So the generic name would be to show ‘missing’ so that no assumption is made for the value type. 

Note that the index (RowProperties/RowTime) of a time/table is able to show NaN or NaT, so it could be nice to show missing double, duration, and datetime as such.

mreijnde commented:
I guess this is what you had in mind? The display of a series now displays the singleton dimension as it would display the missing value from the given index datatype.  I am not sure if this add much information, as the datatype of the singleton index is not used anyway.

mreijnde commented:
Moved the code to handle missing value as index/column to gettable code. This also fixes bug that table conversion did not work for rowSeries().

Note: the behavior of gettable() is now changed. Conversion to valid variable names for columns is not needed and removed. Numerical column names are now displayed without the "x" prefix.
